scripts/track-users.ts

```diff
@@ -24,7 +24,7 @@ function seasonToMonth(season: string) {
 
 // true if a semester is an "old semester," i.e. if a semester on CoursePlan has already passed in real life
 // the idea is that old semesters and new semesters represent whether users are planning in the future or just uploading courses
-function isOld(semester: FirestoreSemester) {
+function isOld(semester) {
```

Comment:
I don't love the implict `any` type here, we end up getting a lot of **TS7006: Parameter 'xxx' implicitly has an 'any' type**. If we're going to use implicit any, could we add `"noImplicitAny": false` to tsconfig.json? I checked, and doing this only causes minor type errors elsewhere that are easily fixed by replacing `never[]` with a real type in a total of 30 seconds.

Reply:
I'm not seeing that warning, how are you seeing it? Do you have linter settings specific to your IDE?

I think `"noImplicitAny": true` is very useful, and we shouldn't use implicit any. We could add types if there's a warning

Reply:
I was getting errors from typescript in Webstorm.